### PR TITLE
Update README.rst

### PR DESCRIPTION
**Pull request by szuecs**
#1466 update short descriptions

### CODE REVIEW

**Comment by szuecs** [comment]
👍

File: README.rst [hunk 1]
```diff
@@ -27,15 +27,25 @@ Features
...
+* Worker Auto Scaling Group with node pools support
 * Flannel overlay networking
-* Cluster autoscaling (using kube-aws-autoscaler_)
+* Cluster autoscaling (using cluster-autoscaler_)
+* Kubernetes DNS with node-local dnsmasq as daemonset and CoreDNS resolver for cluster.local domain.
 * Route53 DNS integration via `External DNS`_
 * AWS IAM integration via kube2iam_
 * Standard components are installed: kube-dns, heapster, dashboard, node exporter, kube-state-metrics
-* Webhook authentication and authorization (roles "ReadOnly", "PowerUser", "Administrator")
+* Webhook authentication and authorization (roles "ReadOnly", "ProductionDeployer", "Manual", "Emergency", "Administrator")
```
Comment (mikkeloscar) [inline README.rst]: `PowerUser` correct, and `ProductionDeployer` should not be listed there. `Production Deployer` just maps to `ReadOnly` in terms of cluster access.

Comment (mikkeloscar) [reply README.rst]: I updated it.

**Comment by mikkeloscar** [comment]
:+1:

**Comment by njuettner** [comment]
👍

**Pull request merged**